tex/artikel/FSR_Vorstellungsrunde.tex

```diff
@@ -189,6 +196,127 @@ \section{Der Fachschaftsrat~(FSR) Physik stellt sich vor}
+Ihr braucht euch nicht alleine fühlen, eure neuen Kommilitonen, die Übungsgruppenleiter oder (tatsächlich) auch die Profs sind immer für euch da. Vor allem aber auch wir, die Fachschaft. :)
+}
+
+\vspace{-0.2cm}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{r}{0cm}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Marius_M_cut.jpg}
+	\end{wrapfigure}
+}
+{
+Hai na, ich bin Marius und studiere wenn alles gut läuft mittlerweile im Master Physik. Da ich recht neu in der Fachschaft bin habe ich noch nicht wirklich einen Zuständigkeitsbereich, weiß aber wohl die ein oder andere Sache zum Studium zu der ich Auskunft geben kann. Ich wünsche euch einen wundervollen Start ins Studium und schaut mal in den Schlosspark, der ist echt schön! c:
+}
+
+\vspace{-0.18cm}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{l}{0cm}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Sarah.jpg}
+	\end{wrapfigure}
+}
+{
+Hey, ich bin Sarah und studiere jetzt im dritten Semster Physik im 1-Fach Bachelor mit Mathe als Nebenfach. In der Fachschaft bin ich seit meinem Studienbeginn und bin in den Teams für die Fibel, die O-Woche und den Hochschultag. 
+Ich wünsche euch allen viel Spaß im Studium. Lasst euch nicht zu sehr stressen und genießt die Zeit.  
+}
+
+\vspace{-0.15cm}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{r}{0cm}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Anna_K_cut.jpg}
+	\end{wrapfigure}
+}
+{
+Hey, ich bin Anna und studiere seit 2023 1-Fach Bachlor Physik. In der Fachschaft helfe ich bei der Evaluation der Lehre mit. Bei Fragen könnt ihr euch gerne an mich wenden.
+Ich wünsche euch einen guten Start ins Studium!
+}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{l}{0cm}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Jona_cut.jpg}
+	\end{wrapfigure}
+}
+{
+Hi ich bin Jona, ich studiere seit 2023 Physik im 1-Fach Bachelor. Ich bin seit dem in der Fachschaft und für die O-Woche mitverantwortlich. 
+Ich wünsche euch viel Spaß in der O-Woche und im Studium. 
+}
+
+\vspace{0.5cm}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{r}{0cm}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Max_W.png}
+	\end{wrapfigure}
+}
+{
+Moin, ich bin Max und mache momentan meinen Master in Physik. Ich helfe unter anderem bei der O-Wochen Planung und dem Awareness Konzept. Falls ihr Fragen habt kann ich euch immer gerne an Leute weiterleiten, die Antworten haben. Euch allen viel Erfolg beim Physik Studium. 
+}
+
+\vspace{-0.1cm}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{l}{0cm}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Matthias.JPG}
+	\end{wrapfigure}
+}
+{
+Hi, ich bin Matthias und im 3. Semester mit Mathe Nebenfach. Wenn nicht schon in der O-Woche wirst du mich wahrscheinlich  im Winter beim Büchermarkt kennen lernen. :) 
+Wenn du glaubst ich könnte dir weiterhelfen, sprich mich gerne an. Viel Spaß in deinem ersten Semester.  
+}
+
+\fibelvorstellung{
+	\begin{wrapfigure}{l}{0cm}
```

Comment:
```suggestion
	\begin{wrapfigure}{r}{0cm}
```

Soll das hier für die abwechselnd linken und rechten Bilder evtl. ein `{r}` sein?

Reply:
Ne ist wegen des Seitenumbruchs so (s. letzte zugehörige kompilierte Fibel)

Reply:
Weil das ja danach auf der neuen Seite beginnt

Reply:
Da sind drei `{l}` in Folge. Zwei `{l}` würde ich wegen dem Seitenumbruch verstehen, aber 3?

Reply:
Guck mal, wie das auf der neuen (letzten) Seite aussieht 
Am Anfang jeder neuen Seite habe ich die, die nebeneinander sind (nicht untereinander) immer auf jeweils links gemacht.

Reply:
Okay, wenn das Absicht ist, passt das ja